torch/distributed/nn/api/remote_module.py

```diff
@@ -143,6 +158,114 @@ def __init__(
             method = torch.jit.export(method)
             setattr(self, method_name, types.MethodType(method, self))
 
+    def _raise_not_supported(self):
+        raise ValueError("Method not supported for RemoteModule")
```

Comment:
nit: Is it possible to add the method name to this error?

Reply:
Sure. Passed method.__name__ to this function.